nixos/modules/services/networking/croc.nix

```diff
@@ -0,0 +1,78 @@
+{ config, lib, pkgs, ... }:
+let
+  inherit (lib) types;
+  cfg = config.services.croc;
+in
+{
+  options.services.croc = {
+    enable = lib.mkEnableOption "croc relay";
+    ports = lib.mkOption {
+      type = types.listOf types.port;
+      default = [9009 9010 9011 9012 9013];
+      description = "Ports of the relay.";
+    };
+    pass = lib.mkOption {
```

Comment:
Please remove and include `passwordFile` option instead.

Reply:
I've requested this feature upstream: https://github.com/schollz/croc/issues/226

Reply:
password file is now supported by upstream, so this option can be changed to disallow a password in the config